docs/static/includes/interfaces/int.pe.udt.schema1.bicep

```diff
@@ -75,7 +75,7 @@ module <singularMainResourceType>_privateEndpoints 'br/public:avm/res/network/pr
   name: '${uniqueString(deployment().name, location)}-<singularMainResourceType>-PrivateEndpoint-${index}'
   params: {
     // Variant 1: A default service can be assumed (i.e., for services that only have one private endpoint type)
-    privateLinkServiceConnections: [
+    privateLinkServiceConnections: (!empty(privateEndpoint.?manualPrivateLinkServiceConnections) ? [] : [
```

Comment:
Please note, this currently does not cover the suggestion of https://github.com/Azure/bicep-registry-modules/issues/946